### PR TITLE
Clear out even MORE helpdb docs

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

File: base/mmap.jl [hunk 1]
```diff
@@ -94,6 +101,65 @@ else
...
+# Now write the data
+write(s, A)
+close(s)
+
+# Test by reading it back in
+s = open("/tmp/mmap.bin")   # default is read-only
+m = read(s, Int)
+n = read(s, Int)
+A2 = Mmap.mmap(s, Matrix{Int}, (m,n))
+```
```
Comment (MichaelHatherly) [inline base/mmap.jl]: Should this be turned into a doctest? Might need to use `mktemp` instead of hardcoding a file name though.

**Comment by MichaelHatherly** [comment]
Looks like some of the signatures in stdlib will need adjusting: https://travis-ci.org/JuliaLang/julia/jobs/188176839#L745.

**Comment by kshyatt** [comment]
Everyone: please hold off on merging this until https://github.com/JuliaLang/julia/pull/19674 is merged :)

**Comment by KristofferC** [comment]
It now is.

File: base/docs/helpdb/Base.jl [hunk 2]
```diff
@@ -225,6 +225,7 @@ See [`RoundingMode`](@ref) for available rounding modes.
 Float32
 
 """
+<<<<<<< HEAD
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: still being moved, should still be deleted here right?

Comment (tkelman) [inline base/docs/helpdb/Base.jl]: there are 2 or 3 more of these

File: base/array.jl [hunk 3]
```diff
@@ -919,6 +919,11 @@ function =={T<:BitInteger}(a::Array{T,1}, b::Array{T,1})
         :memcmp, Int32, (Ptr{T}, Ptr{T}, UInt), a, b, sizeof(T) * len)
 end
 
+"""
+    reverse(v, start=1, stop=length(v))
+
+Return a copy of `v` reversed from start to stop.
+"""
 function reverse(A::AbstractVector, s=1, n=length(A))
```
Comment (hayd) [inline base/array.jl]: Aside: the arg names do not match and it seems like 'stop' is incorrect terminology here...

File: base/docs/helpdb/Base.jl [hunk 4]
```diff
@@ -557,6 +485,7 @@ Determine whether Julia is running an interactive session.
 isinteractive
 
 """
+<<<<<<< HEAD
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: should be removed from this file

**Comment by kshyatt** [comment]
I think I got all the merge artifacts out. Good to go?

File: doc/src/stdlib/math.md [hunk 5]
```diff
@@ -232,7 +232,6 @@ Base.varm
 Base.middle
 Base.median
 Base.median!
-Base.midpoints
```
Comment (tkelman) [inline doc/src/stdlib/math.md]: this is in deprecated.jl but not actually deprecated, it still has tests even. we should move it to the 0.6 section, make it an actual deprecation, and remove the export and tests

Comment (kshyatt) [inline doc/src/stdlib/math.md]: Do you want me to do that now, or in a separate PR?

Comment (tkelman) [inline doc/src/stdlib/math.md]: shouldn't remove its docs until it's actually deprecated

Comment (kshyatt) [inline doc/src/stdlib/math.md]: OK, fixed?

Comment (tkelman) [inline doc/src/stdlib/math.md]: except for this line?

File: base/mmap.jl [hunk 6]
```diff
@@ -165,6 +231,20 @@ mmap{T<:Array}(file::AbstractString, ::Type{T}, len::Integer, offset::Integer=In
 mmap{T<:Array,N}(::Type{T}, dims::NTuple{N,Integer}; shared::Bool=true) = mmap(Anonymous(), T, dims, Int64(0); shared=shared)
 mmap{T<:Array}(::Type{T}, i::Integer...; shared::Bool=true) = mmap(Anonymous(), T, convert(Tuple{Vararg{Int}},i), Int64(0); shared=shared)
 
+"""
+    Mmap.mmap(io, BitArray, dims, offset)
```
Comment (tkelman) [inline base/mmap.jl]: offset has a default, does dims?

File: base/multimedia.jl [hunk 7]
```diff
@@ -190,6 +233,24 @@ end
...
+richest supported multimedia output for `x`, with plain-text [`STDOUT`](@ref) output as a fallback.
+The `display(d, x)` variant attempts to display `x` on the given display `d` only, throwing
+a `MethodError` if `d` cannot display objects of this type.
+
+There are also two variants with a `mime` argument (a MIME type string, such as
+`"image/png"`), which attempt to display `x` using the requested MIME type *only*, throwing
+a `MethodError` if this type is not supported by either the display(s) or by `x`. With these
+variants, one can also supply the "raw" data in the requested MIME type by passing
+`x::AbstractString` (for MIME types with text-based storage, such as text/html or
+application/postscript) or `x::Vector{UInt8}` (for binary MIME types).
```
Comment (tkelman) [inline base/multimedia.jl]: should text/html and application/postscript be code-highlighted?

File: base/docs/helpdb/Base.jl [hunk 8]
```diff
@@ -218,79 +218,6 @@ See [`RoundingMode`](@ref) for available rounding modes.
...
-the byte representation is different.
-
-**Example**: `B = Mmap.mmap(s, BitArray, (25,30000))`
-
-This would create a 25-by-30000 `BitArray`, linked to the file associated with stream `s`.
-"""
-Mmap.mmap(io, ::BitArray, dims = ?, offset = ?)
-
-"""
     bessely0(x)
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: should delete this one here

File: base/random.jl [hunk 9]
```diff
@@ -1441,6 +1441,13 @@ function randsubseq!(r::AbstractRNG, S::AbstractArray, A::AbstractArray, p::Real
     end
     return S
 end
+
+"""
+    randsubseq!(S, A, p::Real)
```
Comment (tkelman) [inline base/random.jl]: you didn't delete this one from helpdb?

File: base/mmap.jl (hunk 6 above)
Comment (kshyatt) [inline base/mmap.jl]: `dims` does not

File: base/multimedia.jl (hunk 7 above)
Comment (kshyatt) [inline base/multimedia.jl]: Yes, they are in the `reprmime` docstring

**Pull request closed without merging**